newIDE/app/src/MainFrame/Toolbar/index.js

```diff
@@ -83,6 +84,13 @@ export class MainFrameToolbar extends PureComponent<Props, State> {
                       this.props.simulateUpdateAvailable();
                   },
                 },
+                {
+                  label: 'Open command palette',
+                  disabled: false,
+                  click: () => {
+                    this.props.openCmdPalette();
+                  },
```

Comment:
nitpicking: if you don't need an extra closure, just do: 

```suggestion
                  click: this.props.openCmdPalette,
```